src/snappy_device_agents/devices/maas2/maas_storage.py

```diff
@@ -490,7 +503,7 @@ def process_format(self, device):
                     f"label={device['label']}",
                 ]
             )
-        # format block-device
+        # if the device does not have a 'volume' key, it's a block device
```

Comment:
return before this point and you can eliminate the "else" and remove the indentation from here down :)

Reply:
Indeed - this works, integrated.